shared/folders/files/index.js

```diff
@@ -1,14 +1,90 @@
 // @flow
 import React, {Component} from 'react'
 import {connect} from 'react-redux'
-import {Text} from '../../common-adapters'
+import {bindActionCreators} from 'redux'
+import {openInKBFS} from '../../actions/kbfs'
+import {navigateBack} from '../../actions/router'
+import Render from './render'
+import _ from 'lodash'
 
-// import Render from './render'
+import type {Folder} from '../list'
+
+type Props = $Shape<{
+  folder: ?Folder,
+  path: string,
+  username: string,
+  navigateBack: () => void,
+  openInKBFS: (path: string) => void
+}>
+
+type State = {
+  visiblePopupMenu: boolean
+};
+
+class Files extends Component<void, Props, State> {
+  state: State;
+
+  _checkFolderExistance (props) {
+    // TODO (AW): make a more user friendly response for when the folder they were hoping to look at has been removed
```

Comment:
I was thinking of an edge case where in the time between the user clicks the folder in the Folders view but before the Files view is rendered, the folder gets removed from the user's favorites list and therefore is no longer reachable in the current store setup.